Mathlib/Topology/Constructions.lean

```diff
@@ -1459,22 +1345,21 @@ the `f i` as `i : ι` varies is simply the topology on `X` induced by `g : X →
 where `Π i, π i` is endowed with the usual product topology. -/
 theorem inducing_infᵢ_to_pi {X : Type _} (f : ∀ i, X → π i) :
     @Inducing X (∀ i, π i) (⨅ i, induced (f i) inferInstance) _ fun x i => f i x := by
+  letI := ⨅ i, induced (f i) inferInstance
   constructor
   erw [induced_infᵢ]
   congr 1
   funext
-  erw [induced_compose]
+  erw [induced_compose]; rfl
 #align inducing_infi_to_pi inducing_infᵢ_to_pi
 
 variable [Finite ι] [∀ i, DiscreteTopology (π i)]
 
 /-- A finite product of discrete spaces is discrete. -/
 instance PiCat.discreteTopology : DiscreteTopology (∀ i, π i) :=
```

Comment:
I don't know why `mathport` translates `Pi.` to `PiCat.`. Should I revert this?

Reply:
Yes, please. There is a PR to mathport that tries to fix this.

Reply:
I merged that PR right now. So in a couple of hours, this problem shouldn't occur anymore.

Reply:
AFAIR, full cycle takes ~5h.